app/views/fact_values/_fact.html.erb

```diff
@@ -8,14 +8,13 @@
         <% end %>
       </td>
   <% end %>
-  <td>
+  <td class="ellipsis">
     <%= fact_name(fact, parent) %>
   </td>
   <td class="fact-col">
     <% unless fact.compose %>
       <%= link_to trunc_with_tooltip(fact.value, 40),
                   fact_values_path("search" => "facts.#{fact.name} = \"#{fact.value}\""),
-                  :title => _("Show all %{name} facts where they are equal to %{value}") % { :name => fact.name, :value => fact.value },
```

Comment:
IMHO we can expand some space by limiting the actions header size

On Tue, Mar 22, 2016 at 10:55 AM, Marek Hulán <notifications@github.com>
wrote:

> In app/views/fact_values/_fact.html.erb
> <https://github.com/theforeman/foreman/pull/3356#discussion_r56952837>:
>
> >      <%= fact_name(fact, parent) %>
> >    </td>
> >    <td class="fact-col">
> >      <% unless fact.compose %>
> >        <%= link_to trunc_with_tooltip(fact.value, 40),
> >                    fact_values_path("search" => "facts.#{fact.name} = \"#{fact.value}\""),
> > -                  :title => _("Show all %{name} facts where they are equal to %{value}") % { :name => fact.name, :value => fact.value },
>
> while not entirely related to the issue, this was displaying two
> conflicting tooltips, there's not much value in this text anyway so I
> removed it
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3356/files/ca3e90848038f61684825d79a98e071e7f7a792b#r56952837>
>